libs/fetcher.client.js

```diff
@@ -290,7 +290,7 @@ Queue.prototype = {
          * @private
          */
         _constructGetUri: function (uri, resource, params, config) {
-            var query = [], matrix = [], id_param = config.id_param, id_val, final_uri = uri + '/resource/' + resource;
+            var query = [], matrix = [], id_param = config.id_param, id_val, final_uri = uri + '/' + resource;
```

Comment:
Yes, i think so, i was remove the if/else before, but the test did not pass.  here is the log:

```
1) Server Fetcher #middleware #GET should respond to GET api request:
     Error: Fetcher "ock_service" could not be found
      at Function.Fetcher.getFetcher (/Users/lenage/Projects/github/fetchr/libs/fetcher.js:78:19)
      at Function.Fetcher.single (/Users/lenage/Projects/github/fetchr/libs/fetcher.js:175:31)
      at /Users/lenage/Projects/github/fetchr/libs/fetcher.js:147:21
      at Context.<anonymous> (/Users/lenage/Projects/github/fetchr/tests/unit/libs/fetcher.js:272:17)
      at Test.Runnable.run (/Users/lenage/Projects/github/fetchr/node_modules/mocha/lib/runnable.js:233:15)
      at Runner.runTest (/Users/lenage/Projects/github/fetchr/node_modules/mocha/lib/runner.js:387:10)
      at /Users/lenage/Projects/github/fetchr/node_modules/mocha/lib/runner.js:470:12
      at next (/Users/lenage/Projects/github/fetchr/node_modules/mocha/lib/runner.js:312:14)
      at /Users/lenage/Projects/github/fetchr/node_modules/mocha/lib/runner.js:322:7
      at next (/Users/lenage/Projects/github/fetchr/node_modules/mocha/lib/runner.js:257:23)
      at Object._onImmediate (/Users/lenage/Projects/github/fetchr/node_modules/mocha/lib/runner.js:289:5)
      at processImmediate [as _immediateCallback] (timers.js:354:15)

  2) Server Fetcher #middleware #GET should respond to GET api request with custom status code:
     Error: Fetcher "ock_service" could not be found
      at Function.Fetcher.getFetcher (/Users/lenage/Projects/github/fetchr/libs/fetcher.js:78:19)
      at Function.Fetcher.single (/Users/lenage/Projects/github/fetchr/libs/fetcher.js:175:31)
      at /Users/lenage/Projects/github/fetchr/libs/fetcher.js:147:21
      at Context.<anonymous> (/Users/lenage/Projects/github/fetchr/tests/unit/libs/fetcher.js:316:17)
      at Test.Runnable.run (/Users/lenage/Projects/github/fetchr/node_modules/mocha/lib/runnable.js:233:15)
      at Runner.runTest (/Users/lenage/Projects/github/fetchr/node_modules/mocha/lib/runner.js:387:10)
      at /Users/lenage/Projects/github/fetchr/node_modules/mocha/lib/runner.js:470:12
      at next (/Users/lenage/Projects/github/fetchr/node_modules/mocha/lib/runner.js:312:14)
      at /Users/lenage/Projects/github/fetchr/node_modules/mocha/lib/runner.js:322:7
      at next (/Users/lenage/Projects/github/fetchr/node_modules/mocha/lib/runner.js:257:23)
      at Object._onImmediate (/Users/lenage/Projects/github/fetchr/node_modules/mocha/lib/runner.js:289:5)
      at processImmediate [as _immediateCallback] (timers.js:354:15)
......
```

any ideas?